lhotse/cut.py

```diff
@@ -1152,6 +1155,80 @@ def truncate(
             supervisions=sorted(supervisions, key=lambda s: s.start),
         )
 
+    def extend(
+        self,
+        *,
+        duration: Seconds,
+        direction: str = "both",
+        preserve_id: bool = False,
+    ) -> "MonoCut":
+        """
+        Returns a new MonoCut that is an extended region of the current MonoCut.
+
+        Note that no operation is done on the actual features or recording -
+        it's only during the call to :meth:`MonoCut.load_features` / :meth:`MonoCut.load_audio`
+        when the actual changes happen (an extended version of features/audio is loaded).
+
+        .. hint::
+
+            This method extends a cut by a given duration, either to the left or to the right (or both).
+            To "expand" a cut by padding, use :meth:`MonoCut.pad`. To "truncate" a cut, use :meth:`MonoCut.truncate`.
+
+        :param duration: float (seconds), controls the duration of the resulting MonoCut.
```

Comment:
this is unclear whether the cut is extended **by** duration or extended **to** duration